src/VirtoCommerce.CatalogModule.Web/Controllers/Api/CatalogModuleAssociationsController.cs

```diff
@@ -0,0 +1,99 @@
+using System.Net;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+using VirtoCommerce.CatalogModule.Core;
+using VirtoCommerce.CatalogModule.Core.Model;
+using VirtoCommerce.CatalogModule.Core.Model.Search;
+using VirtoCommerce.CatalogModule.Core.Search;
+using VirtoCommerce.CatalogModule.Core.Services;
+
+namespace VirtoCommerce.CatalogModule.Web.Controllers.Api
+{
+    [Route("api/catalog/products/{productid}/associations")]
```

Comment:
Please use product**I**d everywhere - in routing, in variable names

Reply:
Maybe we should not include {productid} in the route as seems it is really used only by Get method.